src/modules/classAwesome.js

```diff
@@ -0,0 +1,63 @@
+export default class Awesome {
+  constructor() {
+    this.data = {};
+    this.record = [];
+  }
+
+  addRecord(chore) {
+    this.data = { chore };
+    this.record.push(this.data);
+    return this.data;
+  }
+
+  // removeRecord(rec) {
+  //   const result = this.record.filter(el => el === rec);
+  // }
+
+  local() {
+    localStorage.setItem('tasks', JSON.stringify(this.record));
+    this.returnInfo();
+  }
+
+  updateInput(value, element) {
+    this.record = JSON.parse(localStorage.getItem('tasks'));
+    this.record.forEach((elementForChore) => {
+      if (elementForChore.chore === element) {
+        elementForChore.chore = value;
+      }
+      localStorage.setItem('tasks', JSON.stringify(this.record));
+    });
+  }
+
+  returnInfo() {
+    const tasks = document.querySelector('#tasks');
+    tasks.innerHTML = '';
+    this.record = JSON.parse(localStorage.getItem('tasks'));
+    if (this.record === null) {
+      this.record = [];
+    } else {
+      let index = 0;
+      this.record.forEach((element) => {
+        index += 1;
+        tasks.innerHTML += `
+        <div id = "organize"${index}>
+        <input type="checkbox">
+        <input type = "text" class = "label-input" value = "${element.chore}">
+        <button class = "li-btn">  </button>
+        </div>
+        `;
+      });
+    }
+  }
+
+  eliminate(chore) {
+    let indexArray;
+    this.record.forEach((element, index) => {
+      if (element.chore === chore) {
+        indexArray = index;
+      }
+    });
+    this.record.splice(indexArray, 1);
+    this.local();
+  }
```

Comment:
- [ ] This function doesn't seem to be working correctly; if I have 3 items and i remove the middle one it instead removes the first one please see the video 👇

![todo](https://user-images.githubusercontent.com/18730106/159997464-ce512a65-22a3-42e7-a1e5-db7ae2112bf0.gif)

*removing item "b" instead removes "a"* please kindly address this issue.